src/main/kotlin/Archives/Archive.kt

```diff
@@ -0,0 +1,8 @@
+package Archives
+
+import Notes.Notes
+
+class Archive(
+    val name: String,
+    var listForNotes: MutableList<Notes>
```

Comment:
🍏  Можно сделать это поле нуллабельным, чтобы не сохранять пустые списки. Но тогда нужно будет сделать проверки на null в местах кода, где идет обращение к этому полю.

```suggestion
    var listForNotes: MutableList<Notes>?
```